apps/checkout-app/README.md

```diff
@@ -1,30 +1,78 @@
-## Getting Started
+# Checkout App
 
-First, run the development server:
+[Saleor App](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts) for payment gateways configuration and layout customisation of [Saleor Checkout](https://github.com/saleor/saleor-checkout/blob/main/apps/checkout/README.md)
+
+## Setup
+
+### Development
+
+Start tunnel, so that Checkout App is available from Internet for webhooks:
+
+```bash
+npx saleor app tunnel 3000
```

Comment:
Why npx and not pnpm?

Reply:
Let's try to keep it consistent across the board - atm as far as I know all our other materials/docs/tools use `npx` (like creating the first demo storefront with CLI). What do you think @jwm0

Reply:
We should encourage installing `saleor cli` globally, the same as you would do when developing Shopify, Stripe apps, etc.